kconfig-hda.sh

```diff
@@ -2,4 +2,4 @@
 KCONFIG_DIR=$(dirname ${BASH_SOURCE[0]})
 echo $KCONFIG_DIR
 make defconfig
-scripts/kconfig/merge_config.sh .config $KCONFIG_DIR/base-defconfig $KCONFIG_DIR/hdaudio-codecs-defconfig
+scripts/kconfig/merge_config.sh .config $KCONFIG_DIR/base-defconfig $KCONFIG_DIR/hdaudio-codecs-defconfig $@
```

Comment:
can we generalize the idea and have additional arguments, e.g. to disable PM and enable debug?

Reply:
@plbossart this feature will provide the feature to add additional kconfig to base kconfig by the user. They can try to use like `../kconfig/kconfig-sof-default.sh ../kconfig/nopm-defconfig` if they would like a sof-default kconfig with nopm to do some test.

Reply:
right, but what I meant is that $@ means 'all the arguments', so you could have more than one kconfig passed as argument